caikit/interfaces/common/data_model/primitive_sequences.py

```diff
@@ -0,0 +1,27 @@
+# Standard
+from typing import List
+
+# Local
+from caikit.core.data_model import DataObjectBase, dataobject
+from caikit.interfaces.runtime.data_model.training_management import RUNTIME_PACKAGE
+
+
+# interfaces required for unions of lists
+@dataobject(RUNTIME_PACKAGE)
+class IntSequence(DataObjectBase):
+    values: List[int]
```

Comment:
To ensure explicit compatibility, let's use `Annotated[List[xxxx]], FieldNumber(1)]` in each of these